[ADMIN] Process updates (#4)
FinOps-Open-Cost-and-Usage-Spec/foundation: contributing.md

**1.1.1.1.	Discussion Topic:** These are support or functionality inquiries that we want to have a record of for future reference. Depending on the discussion, these can turn into "Spec Change" issues.

**1.1.1.2.	Proposal:** Used for items that propose a new ideas or functionality that require a larger discussion. This allows for feedback from others before a specification change is actually written. 

**1.1.1.3.	Spec Change:** These track specific spec changes until they are completed. Each spec change will be placed into a milestone once triaged.

**1.1.1.4. FinOps Use Case Feedback:** Processes for introducing, discussing, and implementing new ideas or changes in specifications, including feedback on FinOps use cases not supported by the current specification.

## 2.	Issue Lifecycle

The issue lifecycle is mainly driven by people assign to the Issue. All issue types follow the same general lifecycle. Differences are noted below.

**2.1.	Issue Creation**
The submitter needs to select one of the issue templates (Discussion Topic, Proposal, Spec Change, or FinOps Use Case Feedback, etc) when submitting an issue and fill out the details in the template.

**2.2.	Triage**

> The Maintainers will TRIAGE the issue and apply additional necessary fields and labels. This includes fields for target release, category, and metadata.

Reviewer: Why is TRIAGE in caps?

## 2.	Issue Lifecycle

The issue lifecycle is mainly driven by people assigned to the Issue. All issue types follow the same general lifecycle. Differences are noted below.

**2.1.	Issue Creation**

The submitter must select one of the issue templates (Discussion Topic, Proposal, Spec Change, or FinOps Use Case Feedback, etc) when submitting an issue and fill out the details in the template.

**2.2.	Triage**

The Maintainers will triage the issue and apply additional necessary fields and labels. This includes fields for target release, category, and metadata.

(If needed) Clean up the title to succinctly and clearly state the issue.

**2.3.	Discussion**

Feedback from the signed member community is expected and needed to ensure proposals and spec changes are sufficiently discussed prior to actioning. These can be brought up for discussion at the weekly member meetings at the appropriate time.

* Whoever is working on an issue should add his/her name to the issue. Issues MUST be linked to related Pull Requests. Use GitHub (Development) feature to link an Issue to a PR.

**2.4.	Issue Closure**

> The Issues can be closed Editors or Maintainers. Issues linked to a merged PR will closed automatically. 

Reviewer 2: ```suggestion
Issues can be closed by Editors or Maintainers. Issues linked to a merged PR will closed automatically.
```

The Working Group uses pull requests to track changes. To submit a change to the specification:

**3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension). Ensure that the group has agreed the creation of a PR based on a particular Issue.

**3.2.** Modify the Specification to Address the Issue. This work can happen outside of the GitHub flow as you and your team may want to do live collaboration on the spec content before converting to GitHub and Markdown. Once mostly aligned within your group, you can convert the content to markdown and submit the change to the topic branch created above.

**3.3.** Submit a Pull Request to merge the change from the topic branch to the 'working_draft' branch (see details below)

> **3.4.** Notify the Maintainers about your pull request and ask for feedback. Address comments on your topic branch until reviewers have approved.

Reviewer 3: I think when PR's go from WIP to normal, members should be notified. There's no reason it has to be routed through maintainers only since 1. it causes delays 2. it's already an approved work item. If it's not something in scope for the current milestone, it shouldn't move out of WIP and likely should not even be a PR. We need the Issue to be the place to drive collaboration and support (and get triaged and prioritized by maintainers).

Author: Thanks, Udam. I see your point. I had a different reading. I will reverse the change.

The Working Group uses pull requests to track changes. To submit a change to the specification:

> **3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension). Ensure that the group has agreed the creation of a PR based on a particular Issue.

Reviewer: s/has agreed the creation of/has agreed to create/ - wording

Reviewer 2: ```suggestion
**3.1.** Create a topic branch prefixed with the issue-id included in the name (e.g. 4-resource-name-dimension). Ensure that the group has agreed to create a PR based on a particular Issue.
```
